tests/modules/teams/conftest.py

```diff
@@ -24,3 +24,20 @@ def team_for_regular_user(db, regular_user, readonly_user):
     db.session.delete(regular_user_team_member)
     db.session.delete(team)
     db.session.commit()
+
+@pytest.yield_fixture()
+def team_for_nobody(db):  # pylint: disable=invalid-name
```

Comment:
About pylint, what do you want me to do? Should I create rcfile or remove this `disable`?